examples/google-analytics/README.md

```diff
@@ -0,0 +1,57 @@
+# Welcome to Remix!
```

Comment:
Could you update this README to match the other examples. Please follow the template: https://github.com/remix-run/remix/blob/main/examples/template/README.md